src/osx/textctrl_osx.cpp

```diff
@@ -336,7 +336,7 @@ void wxTextCtrl::OnDropFiles(wxDropFilesEvent& event)
 
 void wxTextCtrl::OnKeyDown(wxKeyEvent& event)
 {
-    if ( event.GetModifiers() == wxMOD_CONTROL )
+    if ( event.ControlDown() )
```

Comment:
This looks wrong because it means that `Cmd+Shift+C` will now copy text, for example, while we really only want `Cmd+C` do it. I.e. this relaxes the check for the existing keys, in addition to allowing using Shift for Redo.